Sources/tart/Commands/Run.swift

```diff
@@ -495,6 +499,25 @@ struct Run: AsyncParsableCommand {
         continue
       }
 
+      // Support remote VM names in --disk command-line argument
+      if let remoteName = try? RemoteName(diskPath) {
+        let vmDir = try VMStorageOCI().open(remoteName)
+
+        // Unfortunately, VZDiskImageStorageDeviceAttachment does not support
+        // FileHandle, so we can't easily clone the disk, open it and unlink(2)
+        // to simplify the garbage collection, so use the <>
```

Comment:
Can we mount the same disk to two VMs? Maybe only in read-only mode?

Reply:
Yes, but that will violate the OCI cache immutability guarantees that we currently have and still doesn't sound very secure compared to having a separate FS object.